app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post
+
+  def updates_post_counter=(count)
+    posts.update_attribute 'likes_counter', count
+  end
```

Comment:
- [ ] Same here, please try to avoid expecting parameters in models, you can get all the necessary data from the DB directly 😉 
Please use:

```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```

- [ ] OPTIONAL: use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new comment has been saved to database, then make the `update_comments_counter` method private